books_data.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "title": "Emi",
+    "author": "Emi"
+  },
+  {
+    "title": "Harry",
+    "author": "Josh"
+  },
+  {
+    "title": "I can",
+    "author": "Tony"
+  }
+]
```

Comment:
- [ ]  When the book.json is deleted the app shows an error and newly created books are not being saved book.json file instead it is being saved in rental and there is no way to create a rental since the app is trying to load data from boob.json. Please make sure to fix this and also handle the case when one or more of the files are missing (the app should work and show no errors).

<img width="708" alt="Screenshot 2023-11-23 at 12 02 19 PM" src="https://github.com/joramot/oop_school_library/assets/44798044/a6573646-7aa9-4380-acf1-fe848c30f9cd">